eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -48,24 +77,35 @@ jobs:
         parameters:
           DependenciesVariableName: vcpkg.deps
 
-      - template: /eng/pipelines/templates/steps/cmake-build.yml
-        parameters:
-          BuildArgs: $(build.args)
+      - pwsh: |
+           Write-Host ("Powershell Module Path:" + $Env:PSModulePath)
+           Get-Module -ListAvailable
+           Write-Host "##vso[task.setvariable variable=VCPKG_ROOT]$(Agent.TempDirectory)/vcpkg"
+        displayName: "List available Powershell Modules"
 
       - template: /eng/common/TestResources/deploy-test-resources.yml
         parameters:
           ServiceDirectory: '.'
 
+      - pwsh: |
+           Write-Host "##vso[task.setvariable variable=VCPKG_ROOT]$(Agent.TempDirectory)/vcpkg"
+        displayName: "List VCPKG env vars"
+
+      - template: /eng/pipelines/templates/steps/cmake-build.yml
```

Comment:
Please move the cmake-build.yml step back up above `deploy-test-resources.yml` as we shouldn't be spinning up live test resources if the code doesn't even compile.